source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst

```diff
@@ -271,14 +271,14 @@ Requires: Wire stripper, small flat screwdriver (optional), yellow/green twisted
 3. Measure the length required to reach the CAN terminals of the PDP (either of the two available pairs). Cut and strip ~5/16" (~8 mm) off this end of the wires.
 4. Insert the wires into the appropriate color coded CAN terminals on the PDP. You may use either of the Yellow/Green terminal pairs on the PDP, there is no defined in or out.
 
-.. note: The PDP ships with the CAN bus terminating resistor jumper in the “ON” position. It is recommended to leave the jumper in this position and place any additional CAN nodes between the roboRIO and the PDP (leaving the PDP as the end of the bus). If you wish to place the PDP in the middle of the bus (utilizing both pairs of PDP CAN terminals) move the jumper to the “OFF” position and place your own 120 ohm terminating resistor at the end of your CAN bus chain.
+.. note: See the :ref:`CAN Wiring Basics<docs/hardware/hardware-basics/can-wiring-basics:Termination>` if you need to terminate the CAN bus somewhere other than the PDP.
 
 PWM Cables
 ----------
 
 .. image:: images/how-to-wire-a-robot/image17.jpg
 
-.. note:: This section details how to wire the SPARK MAX controllers using PWM signaling. This is a recommended starting point as it is simpler and easier to troubleshoot than CAN operation.
+.. note:: This section details how to wire the SPARK MAX controllers using PWM signaling. Many FRC motor controllers can also be wired using :ref:`CAN<docs/hardware/hardware-basics/can-wiring-basics:CAN Wiring Basics>`.
```

Comment:
I think we should leave in why ZTR is recommending PWM and describe why a team might want CAN. Maybe something like this?

```suggestion
.. note:: This section details how to wire the SPARK MAX controllers using PWM signaling. This is a recommended starting point as it is simpler and easier to troubleshoot than CAN operation. The SPARK MAXs (and many other FRC motor controllers) can also be wired using :ref:`CAN<docs/hardware/hardware-basics/can-wiring-basics:CAN Wiring Basics>` which unlocks some advanced functionality and may simplify wiring when many controllers are used.
```

Reply:
I agree with the thought, but I think that is underselling CAN:
1) The only time it makes wiring more complicate IMO is when you have motor controllers placed all the way out at the end of something hard to get to.
2) The other huge selling point and I think I will separate these is that not only does it give advanced control options it gives a ton of additional reporting and analysis options.

Let me know what you think of the change

Reply:
Your new text seems good to me. But you still have the old shorter text in there as well below it...